ppr-ui/src/components/dialogs/SecuredPartyDialog.vue

```diff
@@ -6,19 +6,24 @@
     attach="#app"
     content-class="secured-party-dialog"
   >
-    <v-card id="secured-party-dialog" class="pl-4 pr-1 pt-7 mt-7">
+    <v-card id="secured-party-dialog" :class="!isDuplicate && !isReview ? 'pl-4 pr-1 pt-7 mt-7' : 'pl-1 pr-1 pt-7 mt-7'">
```

Comment:
You can also combine static and dynamic class attributes since only one css class is dynamic. eg:
`class="pr-1 pt-7 mt-7" :class="!isDuplicate && !isReview ? 'pl-4' : 'pl-1'"`

Reply:
will do, I like that approach